CHANGELOG/CHANGELOG-v0.8.0.md

```diff
@@ -0,0 +1,3 @@
+# v0.8.0
+
+* Add a status message to the `PolicySet`.
```

Comment:
Wow. Not much here, huh? The upgrade to Go 1.18 is meaningful, maybe?

Reply:
Looks like I'm getting outvoted on this one since other PRs are being merged.

Reply:
Sorry 😅 I started approving the others without noticing your comment here.

Reply:
I thought about mentioning the upgrade to Go 1.18, but I thought that was an implementation detail not relevant to the user. I could be persuaded otherwise.  :smile: